### PR TITLE
Added some clarifications to the README

### PR DESCRIPTION
**Pull request by cam-carter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,3 +107,12 @@ Email the generated _coding-challenge.bundle_ file to engineering@wonderschool.c
...
 Thanks for taking the time to do this coding challenge and here's hoping we talk soon!
+
+## Things to note
+
+1. Typespecs for Elixir code is appreciated
+2. TDD is favored if time permits, but given possible time constraints, we understand if this is cut short
+3. Using modern React conventions is preferred, e.g. dumb components, hooks, etc
+4. Please avoid using Phoenix generators for context functionality; we'd rather see your code and your patterns :)
+5. Have fun with it! We understand that coding challenges are a pain, but we really appreciate your participation in this process!
+6. If you have any you any extraneous circumstances, we are more than willing to work with you on having the best possible interview process.
```
Comment (jbrody1) [inline README.md:118]: "questions or special circumstances"? I don't think extraneous is the right word here.

**Review by jbrody1 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -44,6 +44,7 @@ Some things to keep in mind:
 
 * Locked tasks cannot have their completion status toggled
 * Tasks remain locked until all of their dependencies have been completed
+* Marking a task as incomplete should lock the corresponding tasks for which it is a dependency
```
Comment (rvargas) [inline README.md:47]: Good addition. I mean, common sense is a bitch.

Comment (rvargas) [inline README.md:47]: Good addition. I mean, common sense is a bitch.

File: README.md [hunk 3]
```diff
@@ -99,3 +107,12 @@ Email the generated _coding-challenge.bundle_ file to engineering@wonderschool.c
 best to review and respond to submissions within 1-2 business days.
 
 Thanks for taking the time to do this coding challenge and here's hoping we talk soon!
+
+## Things to note
+
+1. Typespecs for Elixir code is appreciated
+2. TDD is favored if time permits, but given possible time constraints, we understand if this is cut short
```
Comment (rvargas) [inline README.md:114]: I would change TDD for `automated testing` or something similar. TDD is a development process (write tests before coding), so I don't think we can evaluate that in the challenge.

File: README.md (hunk 1 above)
Comment (rvargas) [reply README.md:118]: Agree, the whole "If you have any you any extraneous circumstances" line is extraneous redacted :P

**Review by rvargas — CHANGES_REQUESTED** [review]
Great clarifications, just a couple of minor changes to do.

**Pull request merged**